routing_model/src/flows_dst.rs

```diff
@@ -263,10 +261,6 @@ pub struct CheckAndProcessElderChange(pub MemberState);
 
 // CheckAndProcessElderChange Sub Routine
 impl CheckAndProcessElderChange {
-    fn start_event_loop(&self) -> Self {
```

Comment:
it may not be used/tested, but it reflect the diagram, the proper fix would be to add a test for it.

Reply:
Ok, I'll revert this (and the other two instances), but I'll add a TODO comment to remove the allow lint they'll need once we implement tests for them.